### PR TITLE
demo/Aries-Workshop.md tweak for Traction Sandbox update

### PR DESCRIPTION
**Pull request by loneil**
Minor tweak for state names in Tenant UI sandbox that will come with updating UI to switch to 2.0 protocols.
Remove a TODO I don't _think_ is needed

### CODE REVIEW

File: docs/demo/Aries-Workshop.md
Comment (loneil) [inline docs/demo/Aries-Workshop.md:86]: Saw all schema references in this doc already using `H7W22uhD4ueQdGaGeiCgaM:2:student id:1.0.0` (and used that one while testing the instructions) so I don't _think_ this is needed?

Comment (swcurran) [reply docs/demo/Aries-Workshop.md:86]: Sounds like it should be removed.

**Review by swcurran — APPROVED** [review]

**Comment by swcurran** [comment]
Your call if you want to remove the todo before merging. Happy to approve again if you do.

**Comment by loneil** [comment]
> Your call if you want to remove the todo before merging. Happy to approve again if you do.

Yeah it's removed in this PR already, so could merge.

**Review by jamshale — APPROVED** [review]

**Pull request merged**
